### ISSUE TITLE
2013-12-23 Client (fake guild emblem doesnt show above the head at battleground)

### ISSUE THREAD

**Issue by vBrenth** [MStr3am/NEMO#86]
I'll guess this is client issue, maybe can be fixed.
game works perfectly/rewards everything except the emblem above the head when on bg map

when on BG map (the emblem doesnt show) 
http://i.imgur.com/mOdbGKd.jpg

But when i warped to GVG map (the emblem shows) 
http://i.imgur.com/Mj8u2c9.jpg

**Comment by dastgir** [comment]
I have kinda researched on this deeply while making eBG, 
And this is conclusion
1) Old clients can show bg score and guild emblem together..
2) new clients disable emblems when bg mask is send
(Referring to maptypeProperty packet)

So for new clients, you have to either choose to have bg score or emblem...

**Comment by vBrenth** [comment]
Hope there will be a solution :D looking forward to this.

**Comment by vBrenth** [comment]
Any update on this? I hope there's a solution i kinda need this one badly.

**Comment by MStr3am** [comment]
I have added the patch to upgrade_3_0 branch (will be merged with master soon)

**Comment by vBrenth** [comment]
Woah! You have fixed for this? O my god.

**Comment by vBrenth** [comment]
@MStr3am  Almost works, However i am getting this sword+custom emblem.
http://imgur.com/jVr6DMW

**Comment by vBrenth** [comment]
@MStr3am  please please? (just the alignment of the emblem would be fine.

**Comment by vBrenth** [comment]
@MStr3am  is there any minimal fix on this :D badly needing <3

**Comment by vBrenth** [comment]
Still waiting for this. @MStr3am

**Comment by cassioronchetti** [comment]
Any update on this? The emblem still not align.

**Comment by vBrenth** [comment]
Did you try @4144 version?

**Comment by cassioronchetti** [comment]
Yes I've tried 4144 last version, in old clients (2013-12-30/ 2014-03-05) the emblem still not align. But in new clients for example (2017-07-14) they are align.

Do you still use 2013 clients and have the emblem for BG align?

I like to use old clients to play pre-renewal servers.

I just want the emblem align...

**Comment by vBrenth** [comment]
Nope, I have moved on from older client, You should too.

**Comment by cassioronchetti** [comment]
Yes, I should move on but I don't like the features of the new clients. I like the classic style and old buttons because I am making a pre-renewal server. But I will think about it.
 Thanks vBrenth to answer me!

**Comment by vBrenth** [comment]
Actually true, I still prefer 2013-12 client, so smooth and no bug at all. minimal size for full client but latest GPu makes them pixelated. I hope there's also a fix for that. and @4144 maybe can fix the allignment for 2013 client.

**Comment by cassioronchetti** [comment]
Yes, I think @4144 should help us on that, I already asked him if it was possible to enable Emblem hover for BG for old clients like 2010 - 2012 (because I remember that eAmod used  2010-07-30  which was the most stable client in the past and had the BG emblem feature) and also would be nice to correct the allignment. I know they are old clients, but a lot of people prefer to use old clients for classic ragnarok servers.
